_source/_posts/2020-03-09-what-i-learned-by-working-remotely-net.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: blog_post
+title: "What I Learned After Working Remotely for a Year"
+author: quorralyne
+description: "Have to work from home suddenly? After moving from an office to a remote dev team, @quorralyne has some insight to share in her first year with Okta."
+tags: [remote, remotework, workfromhome, remotedeveloper, workingremote, workremotely, development]
+tweets:
+- "What is it like to move from working on-site to an online-only team? @quorralyne has experience."
+- "Check out @quorralyne's review of working remotely for the first time."
+image: blog/working-remotely/workingremoteBW.jpg
+---
+
+{% img blog/working-remotely/workingremotefrontface.jpg alt:"Working remote on the sidewalk" width:"500" %}
+
+What happens when you suddenly need to work from home due to a myriad of reasons? Whether it's the daily expectation or an occasional occurrence, working remotely has it's benefits and challenges. After coming to a fully remote team at Okta, I discovered a few things about the reality of working outside of an office building that I'd like to share.
```

Comment:
I would move the image after this paragraph and add `{: .center-image }` to center it.